pkg/scaffold/resource/resource_test.go

```diff
@@ -71,24 +71,27 @@ var _ = Describe("Resource", func() {
 			Expect(instance.Validate().Error()).To(ContainSubstring("kind cannot be empty"))
 		})
 
-		It("should fail if the Kind is not pascal cased", func() {
-			// Base case
-			instance := &Resource{Group: "crew", Kind: "FirstMate", Version: "v1"}
+		It("should succeed if the Kind is valid, according to core Kubernetes", func() {
+			instance := &Resource{Group: "crew", Kind: "fIRstMate", Version: "v1"}
 			Expect(instance.Validate()).To(Succeed())
+		})
 
-			// Can't detect this case :(
-			instance = &Resource{Group: "crew", Kind: "Firstmate", Version: "v1"}
-			Expect(instance.Validate()).To(Succeed())
+		It("should fail if Kind is too long", func() {
+			var kind string
 
-			instance = &Resource{Group: "crew", Kind: "firstMate", Version: "v1"}
-			Expect(instance.Validate()).NotTo(Succeed())
```

Comment:
should bring these tests back though, so we have a full range

Reply:
Done ✅

Reply:
table is fine :-)